lucien-ios/APIClient/LucienAPIClient.swift

```diff
@@ -93,4 +93,59 @@ final class LucienAPIClient: APIClient {
             }
         }
     }
+
+    func createPhotoURL(completion: @escaping (Result<AmazonS3ImageURL>) -> Void) {
+        let lucienRequest = LucienRequest.createPhotoURL
+        let urlRequest = lucienRequest.urlRequest
+        sendRequest(urlRequest) { response in
+            switch response {
+            case .success(let result):
+                guard let result = result else {
+                    return completion(.failure(LucienAPIError.noResult))
+                }
+                let decoder = JSONDecoder()
+                decoder.dateDecodingStrategy = .iso8601
+                guard let s3ImageURL = try? decoder.decode(AmazonS3ImageURL.self, from: result) else {
+                    return completion(.failure(LucienAPIError.cannotDecode))
+                }
+                completion(.success(s3ImageURL))
+            case .failure(let error):
+                completion(.failure(error))
+            }
+        }
+    }
+
+    // TODO: Replace Result<Error> with Result<Comic> when Comic model is complete.
+    func addComicBook(comicTitle: String,
+                      storyTitle: String,
+                      volume: String?,
+                      issueNumber: String?,
+                      publisher: String?,
+                      releaseYear: String?,
+                      comicPhotoURL: String?,
+                      returnDate: String?,
+                      condition: String?,
+                      genre: String?,
+                      completion: @escaping (Result<Error?>) -> Void) {
+        let lucienRequest = LucienRequest.addComicBook(comicTitle: comicTitle,
+                                                       storyTitle: storyTitle,
+                                                       volume: volume,
+                                                       issueNumber: issueNumber,
+                                                       publisher: publisher,
+                                                       releaseYear: releaseYear,
+                                                       comicPhotoURL: comicPhotoURL,
+                                                       returnDate: returnDate,
+                                                       condition: condition,
+                                                       genre: genre)
+        let urlRequest = lucienRequest.urlRequest
+        sendRequest(urlRequest) { response in
+            print(response)
+            switch response {
+            case .success(_):
```

Comment:
Remove the `(_)` for now, you can put it back in with the comic at a later date. Rationale: even code that will eventually be updated should still adhere to the style guide since priorities may change and the update may not happen.